src/API/WesternStatesWater.WestDaat.Client.Functions/System.cs

```diff
@@ -10,135 +9,60 @@
 
 namespace WesternStatesWater.WestDaat.Client.Functions
 {
-    public class System : FunctionBase
+    public class System(ISystemManager systemManager, INotificationManager notificationManager)
+        : FunctionBase
     {
-        public System(ISystemManager systemManager, INotificationManager notificationManager, ILogger<System> logger)
+        [Function(nameof(GetDashboardFilters))]
+        public async Task<HttpResponseData> GetDashboardFilters([HttpTrigger(AuthorizationLevel.Anonymous, "get",
+                Route = "system/filters")]
+            HttpRequestData req)
         {
-            _systemManager = systemManager;
-            _notificationManager = notificationManager;
-            _logger = logger;
-        }
-
-        private readonly INotificationManager _notificationManager;
-        private readonly ISystemManager _systemManager;
-        private readonly ILogger _logger;
-
-        [Function(nameof(GetBeneficialUses))]
-        public async Task<HttpResponseData> GetBeneficialUses([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "system/beneficialuses")] HttpRequestData req)
-        {
-            _logger.LogInformation("Getting Beneficial Uses");
-
-            var results = await _systemManager.GetAvailableBeneficialUseNormalizedNames();
-
-            // This cannot run thru CreateOKResponse because the serializer options will convert enum to string
+            var results = await systemManager.LoadFilters();
```

Comment:
should we keep a log statement in here like was in previously? "getting filters..."